### PR TITLE
feat: add feat edit hero banner

### PR DESCRIPTION
**Pull request by adzharamrullah**
## Changes

- feat: add feat edit hero banner

## Evidence
title: feat: add feat edit hero banner
project: Desa Digital
participants: @adzharamrullah @ayocodingit @Ibwedagama @bangunbagustapa @maulanayuseph

### CODE REVIEW

File: components/HeroBanner/Edit/index.vue [hunk 1]
```diff
@@ -0,0 +1,271 @@
...
+        (this.fileImage || this.infoImage.image)
+      ))
+    }
+  },
+  watch: {
+    show: {
+      handler (val) {
+        this.mShow = val
+        const isActive = this.item.is_active ?? ''
+        this.form = { ...this.item, is_active: isActive.toString() }
```
Comment (Ibwedagama) [inline components/HeroBanner/Edit/index.vue:165]: ask/discussion

hello @adzharamrullah , I just wondering what is the data type of `const isActive` ? is it a `String` or should it be a `Boolean`?

Comment (adzharamrullah) [reply components/HeroBanner/Edit/index.vue:165]: hello @Ibwedagama 
this should be a string, that's why i add method `toString()` to variable `is_active`
yes you must be confused with the name of variable, the problem is naming property in backend `JSON` i thinks this should be named property `status`

Comment (Ibwedagama) [reply components/HeroBanner/Edit/index.vue:165]: oh I see, it's a property from the backend response. Okay got it 👍🏼 @adzharamrullah

**Review by Ibwedagama — APPROVED** [review]

File: components/HeroBanner/Edit/index.vue (hunk 1 above)
Comment (ayocodingit) [reply components/HeroBanner/Edit/index.vue:165]: ask @adzharamrullah 
 
`is_active` is mandatory, why is there a possible `null` value

**Review by bangun-jds — APPROVED** [review]

**Review by maulanayuseph — APPROVED** [review]

File: components/HeroBanner/Edit/index.vue (hunk 1 above)
Comment (adzharamrullah) [reply components/HeroBanner/Edit/index.vue:165]: logicly is never be `null` if existing data from response backend not null, and user cannot submit if data `is_active` is `null`

**Review by ayocodingit — APPROVED** [review]

File: components/HeroBanner/Edit/index.vue (hunk 1 above)
Comment (ayocodingit) [reply components/HeroBanner/Edit/index.vue:165]: oh okey thank you

**Pull request merged**